pkg/cmd/version/version.go

```diff
@@ -35,18 +37,27 @@ const devVersion = "dev"
 const latestReleaseURL = "https://api.github.com/repos/tektoncd/cli/releases/latest"
 
 // Command returns version command
-func Command() *cobra.Command {
+func Command(p cli.Params) *cobra.Command {
 	var check bool
 
 	var cmd = &cobra.Command{
 		Use:   "version",
-		Short: "Prints version information",
+		Short: "Prints tkn and server version information",
```

Comment:
Maybe change comment to pipeline version/update docs as well?

Reply:
I may just revert this back, in case we have more stuff to add and don't end up to write a very long 'Short' version message